### ISSUE TITLE
[Accessibility] 9xMissing alternative text & 2xMissing form label - Login page

### ISSUE THREAD

**Issue by cmunteanuRW** [Talleh/VolleyballPub#4]
**Steps to reproduce:**

https://github.com/user-attachments/assets/0854c1b5-e8c6-4bb8-a636-0a188084d152

**Actual Result:**

![Image](https://github.com/user-attachments/assets/66de94f0-c991-4e44-be48-63eea1d9b27e)

**Expected Result:**
No accessibility issues are present
